### DISCUSSION TITLE
Wrong url while using `Static File Provider`

### DISCUSSION THREAD

**Discussion by datnt97** [OrchardCMS/OrchardCore#12773] [Q&A] [answered]
## The situation: 
- I'm using OrchardCore - RC 2.
- I enabled `Static File Provider` feature. 
- The result is: 
  - It doesn't works in `localhost:44300/robots.txt`.
  ![image](https://user-images.githubusercontent.com/59597401/200272469-f996bad4-d6fa-42fd-8e99-6002cc977f83.png)

  - And works in `localhost:44300///robots.txt` or mulitple slash @@
![image](https://user-images.githubusercontent.com/59597401/200271907-07c87b27-3818-4637-8ef5-cf0263beb0de.png)

## The Question: 
 - How to fix this problem?

**Comment by hishamco** [comment, marked as answer]
Please upgrade to `1.5` then try to enable the same feature
